lib/krikri/harvesters/fedora_harvester.rb

```diff
@@ -0,0 +1,158 @@
+require 'uri'
+require 'async_uri_getter'
+
+module Krikri::Harvesters
+  ##
+  # A harvester implementation for Fedora 3
```

Comment:
I think we should specify that this is specifically for UVA's Fedora 3 instance.